### ISSUE TITLE
setup.py install_requires doesn't match requirements.txt

### ISSUE THREAD

**Issue by firesock** [tintinweb/scapy-ssl_tls#79]
pip install is pulling scapy 2.3.3, which doesn't work, leading to the same behaviour as #69

```
$ pip install --user scapy-ssl_tls 
Collecting scapy-ssl_tls
  Downloading scapy-ssl_tls-1.2.3.tar.gz (117kB)
    100% |████████████████████████████████| 118kB 1.0MB/s 
Collecting scapy (from scapy-ssl_tls)
  Downloading scapy-2.3.3.tgz (1.4MB)
    100% |████████████████████████████████| 1.4MB 234kB/s 
Requirement already satisfied (use --upgrade to upgrade): pycrypto in /usr/lib64/python2.7/site-packages (from scapy-ssl_tls)
Collecting tinyec (from scapy-ssl_tls)
  Downloading tinyec-0.3.1.tar.gz
Installing collected packages: scapy, tinyec, scapy-ssl-tls
  Running setup.py install for scapy
  Running setup.py install for tinyec
  Running setup.py install for scapy-ssl-tls
Successfully installed scapy scapy-ssl-tls tinyec
```

**Comment by tintinweb** [comment]
Thank you for reporting this issue. `setup.py` indeed does not specify any upper bounds for `scapy`. I'll look into that and update this issue.

**Comment by tintinweb** [comment]
#81 will be released as v1.2.3.1 to pypi once testing is finished. Thanks for reporting this to us. 👍
